### ISSUE TITLE
Column width sometimes changes after deleting columns

### ISSUE THREAD

**Issue by craigbrown** [EPPlusSoftware/EPPlus#480]
I have a worksheet where columns `C:E` have a custom width of 18. If I delete column D using EPPlus, column E (which is now moved over to column D) changes its width to the default rather than 18.

Strangely, if I get any of columns C, D or E as an object before the deletion (e.g. by doing `wks.Column(4)`), this issue doesn't occur.

Here's a unit test with an example workbook. This test currently fails, but if you uncomment the commented out line, it passes.

```csharp
[TestMethod]
public void TestColumnWidthsAfterDeletingColumn()
{
    using (var pck = OpenPackage("TestColumnWidths.xlsx"))
    {
        // Get the worksheet where columns 3-5 have a width of around 18
        var wks = pck.Workbook.Worksheets["Sheet1"];
        
        // Check the width of column 5
        //Assert.AreEqual(18.77734375, wks.Column(5).Width, 1E-5);
        
        // Delete column 4
        wks.DeleteColumn(4);
        
        // Check width of column 5 (now 4) hasn't changed
        Assert.AreEqual(18.77734375, wks.Column(4).Width, 1E-5);
    }
}
```
[TestColumnWidths.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/7117264/TestColumnWidths.xlsx)

**Comment by JanKallman** [comment]
I'll have a look.

**Comment by JanKallman** [comment]
ColumnMax is not set correctly when deleting within a column span (ColumnMin-ColumnMax).

**Comment by JanKallman** [comment]
Fixed in version 5.7.5
